### ISSUE TITLE
Multiple kindle emails per calibre-web account

### ISSUE THREAD

**Issue by ocangelo** [janeczku/calibre-web#3027]
**Is your feature request related to a problem? Please describe.**
Family uses multiple kindles, would be nice to be able to manage them with a single account in calibre-web without having to switch all the time

**Describe the solution you'd like**
have a drop down when you click the send button listing:
- all registered kindle emails (so you can pick and choose a single kindle to send to)
- a send to all button
- a send to email button (to send to a generic email)

**Describe alternatives you've considered**
sending to an email alias that has rules/filter to forward all emails to all kindle emails...

**Additional context**
nothing

**Comment by OzzieIsaacs** [comment]
There is an PR addressing this: https://github.com/janeczku/calibre-web/pull/2479

**Comment by ocangelo** [comment]
> There is an PR addressing this: #2479

Doesn't look like there was any progress on that merge for years?

Also looking at the code that sends to the entire list of emails all the time, I'm suggesting a drop down to pick which one you want to send to

**Comment by OzzieIsaacs** [comment]
It's merged

**Comment by ocangelo** [comment]
> It's merged

thanks for the update, it's a great step in the right direction but i still feel like this request is separate.

that change would send to all emails all the time, what i am suggesting here is to have a drop down so you can pick and choose

**Comment by orcofthedarkness** [comment]
@ocangelo Does this work for you?
Just updated to v0.6.23 
![image](https://github.com/user-attachments/assets/cfd19827-83c1-428a-b9db-cf19c05c2991)
and tried to add multiple addresses...
![image](https://github.com/user-attachments/assets/9a7ddb0e-0679-44a1-8387-1b27c824800e)
as soon as I hit save, one the last e-mail address is saved.

**Comment by ocangelo** [comment]
@orcofthedarkness i have the same issue, only the last address is saved on 0.6.23

**Comment by cardonator** [comment]
I agree that we need a way to select which device to send to. I'm not sure how tightly coupled this is to Calibre's way of storing the data, but it would be even better if I could define a list of devices for each user and have each of those devices have their own email to send to. Even better if the users could manage that list themselves instead of being up to the instance admin! (This is basically the only feature Kavita has that makes it better than CW)

**Comment by Orybon** [comment]
Other use case : I'am a dad and i want to send ebook to my kindle or to the kindle of one of my kids. (They are not in age to use calibre-web).

**Comment by ocangelo** [comment]
> > It's merged
> 
> thanks for the update, it's a great step in the right direction but i still feel like this request is separate.
> 
> that change would send to all emails all the time, what i am suggesting here is to have a drop down so you can pick and choose.
> 
> i think we should remove the "Fixed In Nightly" label?

@OzzieIsaacs i think we need to remove the "fixed in nightly" label here?

**Comment by mazaleyrat** [comment]
+1 with the use case from @Orybon : the possibilty to send a book to another user.

**Comment by Orybon** [comment]
@OzzieIsaacs  You closed this one, so should i open another to ask for an enhancement like https://github.com/gelbphoenix/autocaliweb/issues/51 did for autocaliweb ?
